BATCH_MANUAL.md

```diff
@@ -0,0 +1,93 @@
+# LDAR-Sim Batch Running Manual
+
+The batch runner offers an alternative method for executing LDAR-Sim. With this feature, users can run the standard simulation in multiple batches, consolidating the outputs of these batches into a single averaged value whenever applicable
+
+## How to use
+
+To utilize the batch script, follow these steps to configure LDAR-Sim with all parameters in a designated directory. Then, employ the following command line structure:
+```ldar_sim_batchrun.py --n_rep {number_of_repetitions} -P {directory_path}```
+
+Example command to run 100 batch simulations contained in the folder simulations would be:
+```ldar_sim_batchrun.py --n_rep 100 -P simulations```
+
+## Outputs
+
+### Regular Outputs
+
+The following files are maintained for each of programs of the first batch of simulations
+```timeseries_output_{simulation}_{Program name}.csv```
+```leaks_output_{simulation}_{Program name}.csv```
+
+The following file will be kept for the very last batch of simulations that are run.
+```sites_output_{simulation}_{Program name}.csv```
+
+### Sites
+
+`sites_output_{program}_concat.csv`
+
+|Column Names      |Unit   |Description                |
+|------------------|-------|---------------------------|
+|facility_ID       |N/A    |Facility ID of the site|
+|Cum_leaks         |N/A    |Average cumulative number of leaks at the site|
+|init_leaks        |N/A    |Average number of initial leaks found at the site|
+|total_emissions_kg|kg     |Average total emissions per site (in kilograms)|
+|subtype_code      |N/A    |The subtype code of the site|
+|lat               |Degrees|Latitude of the site|
+|lon               |Degrees|Longitude of the site|
+|equipment_groups  |N/A    |Number of equipment groups at the site|
+|n_sims            |N/A    |Number of simulations the site participated in|
+
+```sites_summary.csv```
+
+|Column Names                       |Unit       |Description                |
+|-----------------------------------|-----------|---------------------------|
+|Program                            | N/A       |Name of the program             |
+|Mean_emissions_per_site            |per site   |Mean emissions per site in kg|
```

Comment:
Clarity: Can we adjust the wording to be more clear that this is the mean emissions for a single site, calculated across all sites here. I know we still want to be concise, but since this is documentation meant solely to explain this I think it makes to be extra thorough with our explanations.